### DISCUSSION TITLE
Week 6 Post

### DISCUSSION THREAD

**Discussion by Skye404** [USU-ITLS-x265-23-summer/class-repo#133] [General]
In all honesty, I started feeling nerdy this week. I coded on my own! Granted, the instructions really guided me along, but I did many steps by myself without help in the Colored Marker activity. I was proud of myself because this class has been great exercise for my brain, but things are finally sorta sticking, so I'm able to do some cool stuff now with my code on my own. I thought the instructions were helpful and with the background knowledge from previous weeks, I was able to make connections and understand the instructions this week much more clearly. I also enjoyed connecting what I learned about RGB from my Graphic Design class with the Marker activity this week.

Mihai's experience reminded me of a lot of what I've learned in the past about accessibility. It was good to be reminded of the importance of making things accessible to others. I think it's super cool that he was able to do so much and make such an impact even though he suffered in silence through all his hard work. In several ways, trying to figure out how coding works to fixing bugs is suffering at least on a small degree. It's important to fix the bugs and make webpages easily accessible to users.

Instructional Approaches: clear instructions, provide examples, add color and work with contrast to draw a user's eye to a specific button/location, and embed videos when applicable to the content. 

Project Ideas:
1. Instructional webpage that teaches a lesson on story elements (plot, characters, setting, theme, etc.). I expect to encounter many challenges simply remembering what elements to use to code exactly what I want. I think Google and YouTube will help me as well as the Slack group and the class discussion board. I've also got a lot of notes to turn to, especially when adding color to the website.

2. Create a website portfolio for writing and designs. It may be challenging when trying to make this website easily accessible to viewers. It should look professional, and I am unsure if I can create something that looks professional. I will also turn to Google, my notes, and the class discussion board for help. I will attend office hour eventually, which will also help.

**Comment by metasean** [comment]
> In all honesty, I started feeling nerdy this week. I coded on my own! Granted, the instructions really guided me along, but I did many steps by myself without help in the Colored Marker activity. I was proud of myself because this class has been great exercise for my brain, but things are finally sorta sticking, so I'm able to do some cool stuff now with my code on my own. I thought the instructions were helpful and with the background knowledge from previous weeks, I was able to make connections and understand the instructions this week much more clearly. 

I have tears of joy in my eyes!

> I also enjoyed connecting what I learned about RGB from my Graphic Design class with the Marker activity this week.

Double win!!! 🙌 🚀

> Mihai

I'm also saddened that he had to suffer as he did, but I'm also glad that programming and the web enabled allowed him to contribute and make both his and our world better in the process.
